commandapi-core/src/main/java/dev/jorel/commandapi/CommandAPI.java

```diff
@@ -229,7 +229,9 @@ public void onPlayerQuit(PlayerQuitEvent e) {
 				}
 	
 			}, plugin);
-		}
+			logNormal("Chat preview enabled");
+		} else
+			logNormal("Chat preview is not available");
```

Comment:
I highly discourage the use of if/else statements without braces. The following should be used instead:

```java
} else {
    logNormal("Chat preview is not available");
}
```